src/game/gui/menu.c

```diff
@@ -110,7 +110,11 @@ static void menu_render(component *c) {
             if(m->help_bg2) {
                 video_draw(m->help_bg2, m->help_x - 8, m->help_y - 8);
             }
-            text_render(&m->help_text_conf, TEXT_DEFAULT, m->help_x, m->help_y, m->help_w, m->help_h, (*tmp)->help);
+            // TODO: The text_cache should move into inner component. In this case the one of the s->objs.
```

Comment:
I believe this comment is wrong and it's not invalidated every frame

Reply:
I think what I ran into was that the menu didn't animate, so I called all these dynamic.